src/eko/scale_variations/expanded.py

```diff
@@ -201,7 +201,7 @@ def non_singlet_variation_qed(gamma, a_s, a_em, alphaem_running, order, nf, L):
     sv_ker = non_singlet_variation(gamma[1:, 0], a_s, order, nf, L)
     if alphaem_running:
         if order[1] >= 2:
-            sv_ker += a_em * variation_as1(gamma[0, 1:], L)
+            sv_ker -= a_em * variation_as1(gamma[0, 1:], L)
```

Comment:
Just for my personal understanding: the minus here is the same as added in https://github.com/NNPDF/eko/pull/191 ?

Reply:
Honestly I was confused by the minus sign, but since in the qcd case it is there, then I put it.